### PR TITLE
Feature/link state

### PR DESCRIPTION
**Pull request by jgfidelis**

### CODE REVIEW

File: react/components/Product/ProductCustomizerWrapper.js [hunk 1]
```diff
@@ -109,22 +110,60 @@ class ProductCustomizerWrapper extends Component {
...
+    if (isToggleChoice) {
+      return 'TOGGLE'
+    }
+    return 'MULTIPLE'
+  }
+
   getAssemblyOptions = () => {
     const { chosenAttachments, selectedSku } = this.state
     const { product } = this.props
+    console.log('teste props: ', this.props)
```
Comment (augustolazaro) [inline react/components/Product/ProductCustomizerWrapper.js]: 👀

File: react/components/Product/ProductCustomizerWrapper.js [hunk 2]
```diff
@@ -109,22 +110,60 @@ class ProductCustomizerWrapper extends Component {
...
+    console.log('teste state: ', this.state)
```
Comment (GustavoMA) [inline react/components/Product/ProductCustomizerWrapper.js]: :eyes:

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+## [2.0.5] - 2019-03-21
+
```
Comment (GustavoMA) [inline CHANGELOG.md]: Changelog

File: react/components/Product/ProductCustomizerWrapper.js [hunk 4]
```diff
@@ -200,4 +255,21 @@ class ProductCustomizerWrapper extends Component {
   }
 }
 
-export default injectIntl(withToast(orderFormConsumer(ProductCustomizerWrapper)))
+const withMutation = graphql(
+  gql`
```
Comment (klzns) [inline react/components/Product/ProductCustomizerWrapper.js]: Change to a graphql file

File: react/package.json [hunk 5]
```diff
@@ -1,7 +1,10 @@
 {
   "dependencies": {
     "classnames": "^2.2.6",
+    "graphql": "^14.1.1",
+    "graphql-tag": "^2.10.1",
```
Comment (klzns) [inline react/package.json]: Remove

**Pull request merged**